packages/code-du-travail-data/dataset/fiches_service_public/format.js

```diff
@@ -1,4 +1,4 @@
-const parseReference = require("./parseReference");
+const parseReference = require("./parse-reference");
```

Comment:
why did you rename file here ? most of the filename are camelCase

Reply:
Consistency, some files were camelCased some others not so I picked the most common (in this folder). I always prefer files not to be camel cased though. Old habitude, I find it slighty more readable, and by convention "-" means "these are two distinct words".